runtime/ms-rest-azure/package.json

```diff
@@ -28,7 +28,7 @@
     "async": "0.2.7",
     "uuid": "^3.0.1",
     "adal-node": "^0.1.17",
-    "ms-rest": "^2.2.2",
+    "ms-rest": "^2.2.1",
```

Comment:
why was this changed to 2.2.1? We intend to publish this with new version as the min dependency.

Reply:
good catch -- had that to get tests passing, verified that CI passes (https://travis-ci.org/Azure/azure-sdk-for-node/jobs/270179285) .. Will revert this now.